MMR.UI/Forms/ToggleTricksForm.cs

```diff
@@ -15,23 +15,57 @@ namespace MMR.UI.Forms
     public partial class ToggleTricksForm : Form
     {
         public List<string> Result { get; private set; }
+        public LogicFile lines { get; private set; }
```

Comment:
My only issue is with this change. I was lazy and didn't rename the variable earlier, but can you please rename it to `LogicFile`? I will look at merging this in the new few days hopefully. Thanks :)